pmlc/rt/runtime.cc

```diff
@@ -8,3 +10,22 @@ extern "C" void plaidml_rt_trace(const char *msg) {
   llvm::outs() << msg << "\n";
   llvm::outs().flush();
 }
+
+namespace {
+template <typename T>
+void plaidml_rt_copy(StridedMemRefType<T, 0> *dest,
+                     StridedMemRefType<T, 0> *src, int32_t count) {
+  auto destPtr = dest->data + dest->offset;
+  auto srcPtr = src->data + src->offset;
+  size_t bytes = count * sizeof(T);
+  memcpy(destPtr, srcPtr, bytes);
+}
+} // namespace
+
+extern "C" void plaidml_rt_copy_f32(size_t destRank,
+                                    StridedMemRefType<float, 0> *dest,
```

Comment:
Windows apparently wants these to be rank 1 at a minimum.  Shouldn't actually matter here since we never access any of the per-rank array elements.